odoo_backup_sh/controllers/main.py

```diff
@@ -236,6 +236,14 @@ def fetch_dashboard_data(self):
         })
         return dashboard_data
 
+    @http.route('/odoo_backup_sh/open_iap_recharge', type='http', auth="user")
+    def open_iap_recharge(self, **kwargs):
+        data = {'params': {'user_key': BackupController.get_config_values(
+            'options', ['odoo_backup_user_key'])['odoo_backup_user_key']}}
+        response = requests.post(BACKUP_SERVICE_ENDPOINT + '/get_credit_url', json=data).json()
```

Comment:
I'd prefer to redirect to a link in BACKUP_SERVICE_ENDPOINT and then redirect to odoo.com, rather than making server-to-server request: the user will see the message "Within seconds you will be redirected..." quicker

Reply:
ok